.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
             source /usr/local/share/virtualenvs/tap-hubspot/bin/activate
             # BUG https://jira.talendforge.org/browse/TDL-15447
             echo "Disabling the following $PYLINT_DISABLE_LIST"
-            pylint tap_hubspot -d "$PYLINT_DISABLE_LIST,consider-using-from-import,logging-format-interpolation,unused-variable,singleton-comparison,useless-object-inheritance,unnecessary-pass,wrong-import-order,global-statement,unused-argument,unused-import,wildcard-import,unused-wildcard-import,import-error,use-list-literal,no-self-use,duplicate-code,bad-whitespace,bad-continuation"
+            pylint tap_hubspot -d "$PYLINT_DISABLE_LIST,duplicate-code,wrong-import-order"
```

Comment:
I think the card suggests fixing the pylint error or adding the remaining error to tap-tester $PYLINT_DISABLE_LIST, so can you please fix or disable `duplicate-code,wrong-import-order` in $PYLINT_DISABLE_LIST.

Reply:
thanks for reviewing @savan-chovatiya , I've removed duplicate from config.yaml and add comment to disable duplicate-code in test_offsets.py. I'm not making any code changes to remove duplication since that is a test case and wrong-import-order i've added it in  $PYLINT_DISABLE_LIST [(PR)](https://github.com/stitchdata/environments/pull/713). I still have it in config.yaml just to make cci pass once environment PR gets approved, i will remove it from config.yaml